source/nvwave.py

```diff
@@ -140,7 +149,29 @@ def _isDebugForNvWave():
 	return config.conf["debugLog"]["nvwave"]
 
 
-class WavePlayer(garbageHandler.TrackedObject):
+class AudioSession(NamedTuple):
+	"""Identifies an audio session.
+	An audio session may contain multiple streams. The guid identifies the
+	session. The name is shown in the system Volume Mixer.
+	"""
+	guid: GUID
+	name: str
+
+
+#: The audio session to use by default.
+defaultSession = AudioSession(
+	GUID("{C302B781-00AF-4ECC-ACB7-7DF16AF7D55E}"),
+	"NVDA"

```

Comment:
Ideally, we'd call this "NVDA speech", since that's mostly what it will get used for. However, I didn't do that because it's possible add-ons output non-speech sound without using playWaveFile or tones (or otherwise setting a session). If this isn't a "real" concern or if the benefits of calling it "NVDA speech" outweigh the potential losses, I can change this.